api/enki.py

```diff
@@ -416,7 +424,7 @@ def checkout(
                     "User validation against Enki server with %s / %s was unsuccessful."
                     % (patron.authorization_identifier, pin)
                 )
-                raise AuthorizationFailedException()
```

Comment:
This was the only place that we used `AuthorizationFailedException`, everywhere else used `PatronAuthorizationFailedException` and it had the same problem detail, so I removed  `AuthorizationFailedException`.